components/ProfilePage/ProfileButtons.tsx

```diff
@@ -76,16 +81,24 @@ export function ProfileButtonsUser({
     onProfilePublicityChanged(!isProfilePublic)
   }
   return (
-    <div className={`d-grid gap-2 col-12 m-3`}>
-      <EditProfileButton className={`py-1`} />
-      <ToggleButton
-        toggleState={isProfilePublic || false}
-        stateTrueLabel={t("forms.makePrivate")}
-        stateFalseLabel={t("forms.makePublic")}
-        onClick={handleSave}
-        className={`py-1`}
-      />
-    </div>
+    <>
+      {isUser ? (
+        <div className={`d-grid gap-2 col-12 m-3`}>
+          <EditProfileButton className={`py-1`} />
+          <ToggleButton
+            toggleState={isProfilePublic || false}
+            stateTrueLabel={t("forms.makePrivate")}
+            stateFalseLabel={t("forms.makePublic")}
+            onClick={handleSave}
+            className={`py-1`}
+          />
+        </div>
+      ) : (
+        <div className={`d-grid gap-2 col-12 m-3`}>
+          <FollowButton profileId={profileId} user={user} />
```

Comment:
This should be FollowOrgButton? Or we could rename this to be more holistic in the FollowButton file

Reply:
There was an old <FollowButton> (components/ProfilePage/FollowButton.tsx) that was obsolete and getting confused with the one from (components/Shared/FollowButton.tsx)

the current button (components/Shared/FollowButton.tsx) consists of a base level component BaseFollowButton which then feeds into:

a. FollowBillButton
b. FollowOrgButton

each uses a slightly different query for Bills or Users* respectively

*Orginally we were only going to allow following orgs but we changed are mind and now there are *some times* legecy references to "Org" that should be read to mean "Users"